setup.cfg

```diff
@@ -60,6 +61,7 @@ console_scripts =
     bumble-usb-probe = bumble.apps.usb_probe:main
     bumble-link-relay = bumble.apps.link_relay.link_relay:main
     bumble-bench = bumble.apps.bench:main
+    bumble-pandora_server = bumble.apps.pandora_server:main
```

Comment:
I recommend using `bumble-pandora-server` (`-` instead of `_`) to be consistent with the other entries.